sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -107,14 +107,14 @@
 
 # Find out the underneath physical port list by logical name
 def logical_port_name_to_physical_port_list(port_name):
-    if port_name.startswith("Ethernet"):
+    try:
+        return [int(port_name)]
```

Comment:
I'm not sure we even need to handle this case. All platforms should provide a port_config.ini (or the new platform.json) file which should specify a logical port name.

@keboliu: This has been present since your initial commit of xcvrd. Can you explain your reasoning for trying to cast the logical port name to an int if the port name doesn't start with "Ethernet"?